src/roman_datamodels/stnode/_node.py

```diff
@@ -56,12 +56,15 @@ def _check_value(value, schema, validator_context):
     temp_schema = {"$schema": "http://stsci.edu/schemas/asdf-schema/0.1.0/asdf-schema"}
     temp_schema.update(schema)
     validator = asdfschema.get_validator(temp_schema, validator_context, validator_callbacks)
-
     validator.validate(value, _schema=temp_schema)
     validator_context.close()
 
 
 def _validate(attr, instance, schema, ctx):
+    if type(instance) == DNode:
```

Comment:
Would you add a comment here about why `isinstance` is not used? This would be important for if/when ruff decides to apply [E721](https://docs.astral.sh/ruff/rules/type-comparison/) and turn this into an `isinstance` check.